geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/records/package.scala

```diff
@@ -13,5 +13,11 @@ package object records {
   object GeometryEncoding extends Enumeration {
     type GeometryEncoding = Value
     val Wkt, Wkb = Value
+    def apply(encodingString: String): GeometryEncoding = {
+      encodingString match {
+        case "Wkb" => GeometryEncoding.Wkb
+        case "Wkt" => GeometryEncoding.Wkt
```

Comment:
might be good to lowercase match, and add a `case _ => throw ` for clarity

Reply:
Good idea.  Can you see if you like how I've updated it?

Reply:
looks good, thanks.